doc/_static/drawer/mpldrawer_examples.py

```diff
@@ -0,0 +1,226 @@
+    drawer.ctrl(4, [1,2])
+
+    drawer.measure(5, 0)
+
+    drawer.fig.suptitle('My Circuit', fontsize='xx-large')
+    plt.savefig(path.join(folder, savefile))
+    plt.style.use('default')
+    plt.close()
+
+def integration_rcParams(savefile="example_rcParams.png"):
+    plt.rcParams['patch.facecolor'] = 'white'
+    plt.rcParams['patch.edgecolor'] = 'black'
+    plt.rcParams['patch.linewidth'] = 2
+    plt.rcParams['patch.force_edgecolor'] = True
+
+    plt.rcParams['lines.color'] = 'black'
+
+    drawer = MPLDrawer(n_wires=5,n_layers=5)
+
+    drawer.label(["0","a",r"$|\Psi\rangle$",r"$|\theta\rangle$", "aux"])
+
+    drawer.box_gate(0, [0,1,2,3,4], "Entangling Layers", text_kwargs={'rotation':'vertical'})
+    drawer.box_gate(1, [0, 1], "U(θ)")
+
+    drawer.box_gate(1, 4, "Z")
+
+    drawer.SWAP(1, (2, 3))
+    drawer.CNOT(2, (0,2))
+
+    drawer.ctrl(3, [1,3], control_values = [True, False])
+    drawer.box_gate(3, 2, "H", zorder=2)
+
+    drawer.ctrl(4, [1,2])
+
+    drawer.measure(5, 0)
+
+    drawer.fig.suptitle('My Circuit', fontsize='xx-large')
+
+
+    plt.savefig(path.join(folder, savefile))
+    plt.style.use('default')
+    plt.close()
+
+def integration_formatted(savefile="example_formatted.png"):
+
+    wire_kwargs = {"color": "indigo", "linewidth": 4}
+    label_kwargs = {"fontsize": "x-large", 'color': 'indigo'}
+    box_kwargs = {'facecolor':'lightcoral', 'edgecolor': 'maroon', 'linewidth': 5}
+    text_kwargs = {'fontsize': 'xx-large', 'color':'maroon'}
+    swap_kwargs = {'linewidth': 4, 'color': 'darkgreen'}
+
+    measure_box = {'facecolor': 'white', 'edgecolor': 'indigo'}
+    measure_lines = {'edgecolor': 'indigo', 'facecolor': 'plum', 'linewidth': 2}
+
+    #########################################################
+
+    drawer = MPLDrawer(n_wires=2,n_layers=4, wire_kwargs=wire_kwargs)
+
+    drawer.label(["0","a"],
+                text_kwargs=label_kwargs)
+
+    drawer.box_gate(0, 0, "Z", box_kwargs = box_kwargs, text_kwargs=text_kwargs)
+
+    drawer.SWAP(1, (0,1), kwargs=swap_kwargs)
+
+    drawer.CNOT(2, (0,1), color='teal')
+
+    drawer.ctrl(3, (0,1), color='black')
+
+    for wire in range(2):
+        drawer.measure(4, wire, box_kwargs=measure_box, lines_kwargs=measure_lines)
+
+    drawer.fig.suptitle('My Circuit', fontsize='xx-large')
+    plt.savefig(path.join(folder, savefile))
+    plt.close()
+
+def float_layer(savefile="float_layer.png"):
+
+    drawer = MPLDrawer(2,2)
+
+    drawer.box_gate(layer=0.5, wires=0, text="Big Gate", extra_width=0.5)
+    drawer.box_gate(layer=0, wires=1, text="X")
+    drawer.box_gate(layer=1, wires=1, text="Y")
+
+    plt.savefig(path.join(folder, savefile))
+    plt.close()
+
+
+labels()
+labels_formatted()
+box_gates()
+box_gates_formatted()
+ctrl()
+CNOT()
+SWAP()
+SWAP_formatted()
+measure()
+measure_formatted()
+integration()
+float_layer()
+integration(style='Solarize_Light2', savefile="example_Solarize_Light2.png")
+integration_rcParams()
+integration_formatted()
```

Comment:
Maybe put these in an `if __name__ == "__main__":` so that they don't get executed if someone attempts to import the examples somewhere (they probably shouldn't, but it could be a bit inconvenient if they did).
```suggestion
if __name__ == "__main__":
    labels()
    labels_formatted()
    box_gates()
    box_gates_formatted()
    ctrl()
    CNOT()
    SWAP()
    SWAP_formatted()
    measure()
    measure_formatted()
    integration()
    float_layer()
    integration(style='Solarize_Light2', savefile="example_Solarize_Light2.png")
    integration_rcParams()
    integration_formatted()
```
```